Assets/HoloToolkit/Utilities/Scripts/Utils.cs

```diff
@@ -10,14 +10,10 @@ namespace HoloToolkit.Unity
     /// </summary>
     public static class Utils
     {
+        [System.Obsolete("Use GameObjectExtensions.SetLayerRecursively(gameObject, layer) instead;")]
```

Comment:
Like your approach of making it obsolete! Let's use it for now and see how it pans out.